### PR TITLE
Add WorkingDir support to quadlet

### PR DESCRIPTION
**Pull request by rhatdan**
One feature needed for podmansh is the ability to set the default homedir to be the workingdir when you login.

#### Does this PR introduce a user-facing change?

```release-note
Add WorkingDir support to quadlet, to specify the default working dir in a container.
```

### CODE REVIEW

File: test/e2e/quadlet_test.go [hunk 1]
```diff
@@ -571,6 +571,7 @@ var _ = Describe("quadlet system generator", func() {
 		Entry("remap-auto2.container", "remap-auto2.container"),
 		Entry("remap-keep-id.container", "remap-keep-id.container"),
 		Entry("remap-keep-id2.container", "remap-keep-id2.container"),
+		Entry("workingdir.container", "workingdir.container"),
```
Comment (Luap99) [inline test/e2e/quadlet_test.go:574]: this file is not commited

**Comment by rhatdan** [comment]
@lsm5 PTAL

**Review by lsm5 — APPROVED** [review]
LGTM

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 2]
```diff
@@ -135,6 +135,7 @@ Valid options for `[Container]` are listed below:
 | Tmpfs=/work                    | --tmpfs /work                                        |
 | User=bin                       | --user bin                                           |
 | UserNS=keep-id:uid=200,gid=210 | --userns keep-id:uid=200,gid=210                     |
+| WorkingDir=$HOME               | --workdir $HOME                                      |
 | VolatileTmp=true               | --tmpfs /tmp                                         |
 | Volume=/source:/dest           | --volume /source:/dest                               |
```
Comment (Luap99) [inline docs/source/markdown/podman-systemd.unit.5.md:140]: W should be below V

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 3]
```diff
@@ -135,6 +135,7 @@ Valid options for `[Container]` are listed below:
 | Tmpfs=/work                    | --tmpfs /work                                        |
 | User=bin                       | --user bin                                           |
 | UserNS=keep-id:uid=200,gid=210 | --userns keep-id:uid=200,gid=210                     |
+| WorkingDir=$HOME               | --workdir $HOME                                      |
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-systemd.unit.5.md:138]: alpha please, here too

File: test/e2e/quadlet_test.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply test/e2e/quadlet_test.go:574]: and please flop at least volume/workingdir

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 4]
```diff
@@ -491,6 +492,12 @@ created by using a `$name.volume` Quadlet file.
 
 This key can be listed multiple times.
 
+### `WorkingDir=` (default to images working dir)
```
Comment (vrothberg) [inline docs/source/markdown/podman-systemd.unit.5.md:495]:
```suggestion
### `WorkingDir=`
```

It confused me a bit.   Absolutely non-blocking.

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**